raster-foundry/raster-foundry
# Promise audit part 1

## Overview
Fix modal based promise issues which create logs

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Symlinks from new migrations present or corrected for any new migrations~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.


## Testing Instructions
 * Verify that modals no longer throw console errors when you close them by clicking the backdrop or dismissing them.
* Verify that other promises which have been updated now handle failures

Closes #4208
